test/test.js

```diff
@@ -1,169 +1,253 @@
-'use strict';
+const chai = require('chai');
 
-var Metromatic = require('../index');
-var chai = require('chai');
-var expect = chai.expect;
-var sinon = require('sinon');
-var sinonChai = require('sinon-chai');
-var EventEmitter = require('events').EventEmitter;
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const { EventEmitter } = require('events');
+const cloudwatchMetrics = require('cloudwatch-metrics');
+const Metromatic = require('../index');
+
+const { expect } = chai;
 
 chai.use(sinonChai);
 
-describe('Metromatic', function () {
-  var object;
+describe('Metromatic', () => {
+  let object;
 
-  function createMetromatic (object, metrics) {
-    var options = {
+  function createMetromatic(emitter, metrics) {
+    const options = {
       statsd: {
         host: 'localhost',
-        port: 8125
+        port: 8125,
       },
-      metrics: metrics
+      backends: [
+        {
+          type: 'cloudwatch', region: 'us', groupName: 'MyService', accessKeyId: 'keyid', secretAccessKey: 'secret',
+        },
+        { type: 'custom', send: () => {} },
+      ],
+      metrics,
     };
 
-    Metromatic.instrument(object, options);
+    Metromatic.instrument(emitter, options);
   }
 
-  beforeEach(function () {
+  beforeEach(() => {
     object = new EventEmitter();
   });
 
-  describe('timed metrics', function () {
-    var clock;
-    var timing;
-
-    beforeEach(function () {
-      if (clock) clock.restore();
-      if (timing) timing.restore();
-    });
-
-    it('reports a timed metric between the start and stop events', function () {
+  describe('setup', () => {
+    it('initializes Cloudwatch metrics', () => {
```

Comment:
These first three test cases are very backend-oriented. Should we have them in `test/backends.js` instead?